server/src/main/scala/com/ossuminc/riddl/lsp/server/RiddlLSPTextDocumentService/DocContentMgmt/ActionsAndLenses.scala

```diff
@@ -64,7 +64,7 @@ object ActionsAndLenses {
       }
       .getOrElse(
         Future
-          .failed(new UnavailableResourceException())
+          .failed(new Throwable("Server not initialized with a doc"))
```

Comment:
Using Throwable is generally unwise. Use Exception if it is recoverable, or RuntimeException if it isn't recoverable